src/tests/util.ts

```diff
@@ -168,3 +177,14 @@ export async function assertHasStackViolation(
     const allViolations = await runStackPolicy(stackPolicy, args);
     assertHasViolation(allViolations, wantViolation);
 }
+
+// Helper to check if `type` is the type of `resourceClass`.
+function isTypeOf<TResource extends Resource>(
```

Comment:
Hmm is this something we need in the policy package?

Reply:
We could definitely consider exposing it in the policy package, to help policy authors write tests that create objects of `ResourceValidationArgs` or `PolicyResource`. And also to encapsulate the details of how we're checking if it's an instance in case we ever have a better way of doing it down the road. Though, in this case, our tests are never calling the `isType`/`asType` methods, so I could have just stubbed them out as unimplemented.